<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1514</width>
    <height>760</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" rowspan="2" colspan="9">
     <layout class="QGridLayout" name="gridLayout"/>
    </item>
    <item row="1" column="1" colspan="3">
     <widget class="QTreeWidget" name="treeWidget">
      <property name="font">
       <font>
        <family>SauceCodePro Nerd Font</family>
       </font>
      </property>
      <property name="columnCount">
       <number>2</number>
      </property>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string notr="true">2</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QPushButton" name="btnRefreshTree">
      <property name="text">
       <string>刷新成员列表-RefreshTree</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="editFind"/>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="btnFindPrevious">
      <property name="text">
       <string>查找上一个-FindPrevious</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="btnFindNext">
      <property name="text">
       <string>查找下一个-FindNext</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QLineEdit" name="editReplace"/>
    </item>
    <item row="2" column="6">
     <widget class="QPushButton" name="btnReplace">
      <property name="text">
       <string>替换-Replace</string>
      </property>
     </widget>
    </item>
    <item row="2" column="7">
     <widget class="QPushButton" name="btnPreviousError">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>&lt;&lt;&lt;&lt;</string>
      </property>
     </widget>
    </item>
    <item row="2" column="8">
     <widget class="QPushButton" name="btnNextError">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>&gt;&gt;&gt;&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
   <widget class="QTextEdit" name="editShowMsg">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>280</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SauceCodePro Nerd Font</family>
     </font>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::NoWrap</enum>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1514</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>文件-File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>工具-Tools</string>
    </property>
    <addaction name="actionGenerate"/>
    <addaction name="separator"/>
    <addaction name="actionCompiling"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Tools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>打开-Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>保存-Save</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>另存-SaveAS...</string>
   </property>
  </action>
  <action name="actionGenerate">
   <property name="text">
    <string>获取-Generate</string>
   </property>
  </action>
  <action name="actionCompiling">
   <property name="text">
    <string>编译-Compiling</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于-About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
